checkbox-provider-ce-oem/units/thermal-sensor/jobs.pxu

```diff
@@ -1,16 +1,23 @@
-id: ce-oem-sensor/thermal
-_summary: Check Thermal sensor
+id: thermal_zones
+plugin: resource
+_summary: Gather list of thermal zone on the platform
+_description:
+ Gather list of all thermal zone on the platform
+estimated_duration: 1s
+command: thermal_sensor_test.py dump
+
+unit: template
+template-resource: thermal_zones
+template-filter: thermal_zones.mode == 'enabled'
```

Comment:
Please change thermal_zones.mode == 'enabled' to required.
It will make it still create the job but mark as skip automatically.

Reply:
per discussion, the test job will failed if the thermal mode is disabled.